src/SprykerEco/Shared/CrefoPayApi/CrefoPayApiConstants.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Shared\CrefoPayApi;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface CrefoPayApiConstants
+{
+    public const CREATE_TRANSACTION_ACTION_URL = 'CREFO_PAY_API:CREATE_TRANSACTION_ACTION_URL';
```

Comment:
If it's crefo pay's urls I'm really not sure if we need them as config values. It's the same thing as a fields that you moved inside the logic.

Reply:
Probably you are right.